tests/php/src/WithoutBlockPreRendering.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Trait WithoutBlockPreRendering.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Tests;
+
+/**
+ * Helper trait to skip block pre-rendering for tests that are not block related.
+ *
+ * @package AmpProject\AmpWP
+ */
+trait WithoutBlockPreRendering {
+
+	/**
+	 * Set up.
+	 */
+	public function setUp() { //phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
+		parent::setUp();
```

Comment:
I've opted for creating the post in `setUp()` because `$post` is reverted to `null` on `tearDown()` when created in `setupBeforeClass()`.